adagrams/game.py

```diff
@@ -1,13 +1,107 @@
+import random
 from random import randint
 
+def establish_pool():
+    pool = []
+    letter_counts= {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+  
+    for key, value in letter_counts.items():
+        for i in range(value):
```

Comment:
Love that you are ensuring that the correct weights per letter are being accounted for with this loop. When we are using a for range loop that doesn't utilizes the loop variable, `i`, we can instead use `_` to signal to other developers and ourselves that this loop is just to run our iteration logic **x** amount of times. 
```suggestion
        for _ in range(value):
```